cylc/flow/scheduler.py

```diff
@@ -1239,6 +1241,25 @@ def run_event_handlers(self, event, reason=""):
             event, str(reason), self.workflow, self.uuid_str, self.owner,
             self.host, self.server.port))
 
+    def release_unheld_tasks(self):
```

Comment:
I find this name quite confusing

Reply:
Picked to follow `release_queued_tasks`, feel free to propose an alternative.